mailersend/emails/emails.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from mailersend.base import base
+
+message = {}
+
+
+class NewEmail(base.NewAPIClient):
+    def __init__(self):
+        baseObj = base.NewAPIClient()
+        super(NewEmail, self).__init__(
+            baseObj.api_base,
+            baseObj.headers_default,
+            baseObj.headers_auth,
+            baseObj.mailersend_api_key,
+        )
+
+    def setMailFrom(self, mail_from, message):
```

Comment:
from and recipients have email and name

Reply:
fields now accept dict and list of dicts respectively